app/src/processing/app/Mode.java

```diff
@@ -678,26 +680,71 @@ public DefaultMutableTreeNode buildExamplesTree() {
       }
 
       // Get examples for third party libraries
-      DefaultMutableTreeNode contributed = new
+      DefaultMutableTreeNode contributedLibExamples = new
         DefaultMutableTreeNode(Language.text("examples.libraries"));
       for (Library lib : contribLibraries) {
         if (lib.hasExamples()) {
-            DefaultMutableTreeNode libNode = new DefaultMutableTreeNode(lib.getName());
+            DefaultMutableTreeNode libNode =
+              new DefaultMutableTreeNode(lib.getName());
             base.addSketches(libNode, lib.getExamplesFolder());
-          contributed.add(libNode);
+          contributedLibExamples.add(libNode);
         }
       }
-      if(contributed.getChildCount() > 0){
-        root.add(contributed);
+      if(contributedLibExamples.getChildCount() > 0){
```

Comment:
Please be mindful of the [Style Guidelines](https://github.com/processing/processing/wiki/Style-Guidelines).